instrumentation/jedis-5.0.0/build.gradle

```diff
@@ -0,0 +1,24 @@
+jar {
+    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.jedis-5.0.0' }
+}
+
+dependencies {
+    implementation(project(":agent-bridge"))
+    implementation(project(":agent-bridge-datastore"))
+    implementation(project(":newrelic-api"))
+    implementation(project(":newrelic-weaver-api"))
+    implementation("redis.clients:jedis:5.0.0")
+}
+
+verifyInstrumentation {
+    passes 'redis.clients:jedis:[5.0.0,6.0.0)'
```

Comment:
why not use
```
passesOnly 'redis.clients:jedis:[5.0.0,)
```
If it works for 6.0.0, then we won't have to change this. If it doesn't, the verify instrumentation will tell us about it.

Reply:
updated!